tuf/data/types.go

```diff
@@ -355,7 +355,7 @@ var defaultExpiryTimes = map[RoleName]time.Duration{
 	CanonicalTimestampRole: notary.Day,
 }
 
-// SetDefaultExpiryTimes allows one to change the default expiries.
+// SetDefaultExpiryTimes allows one to change the default expires.
```

Comment:
expiries is correct

Reply:
Thank you, this has been reverted as well.